src/common/Types.ts

```diff
@@ -463,6 +469,10 @@ export interface ButtonProps extends CommonStyledProps<ButtonStyleRuleSet>, Comm
     disableTouchOpacityAnimation?: boolean;
     activeOpacity?: number;
     underlayColor?: string;
+
+    // Web only.
+    id?: string; // Needed for accessibility to be able to use labelledBy attribute.
```

Comment:
Please correct spelling. Should be "labeledBy".

Reply:
I've addressed the comments and added documentation to respective *.md files

Reply:
In the standard it's double «l» — https://www.w3.org/TR/wai-aria/states_and_properties#aria-labelledby

Reply:
Lets wait with merging it. I think I still need to expose aria-expanded attribute

Reply:
Actually lets go without aria-expanded for now. We can add it later if we see it's needed.